README.md

```diff
@@ -722,7 +722,9 @@ Event                     | Description
 `AFTER_FILE_READ`         | Will be triggered on test files parsing right after reading the file. The handler accepts data object with `file`, `browser` (browser id string) and `hermione` (helper which will be available in test file) fields.
 `AFTER_TESTS_READ`        | Will be triggered right after tests read via `readTests` or `run` methods with `TestCollection` object.
 `RUNNER_START`            | Will be triggered before test execution. If a handler returns a promise, tests will be executed only after the promise is resolved. The handler accepts an instance of a runner as the first argument. You can use this instance to emit and subscribe to any other available events.
-`RUNNER_END`              | Will be triggered after test execution. If a handler returns a promise, tests will be executed only after the promise is resolved. The handler accepts a stats of tests execution.
+`RUNNER_END`              | Will be triggered after test execution. If a handler returns a promise, worker farm will be ends only after the promise is resolved. The handler accepts a stats of tests execution.
```

Comment:
`will be ended`
`accepts an object with tests execution statistics` (if I understood the meaning of previous phrase right)